src/site/content/en/learn/css/box-model/index.md

```diff
@@ -0,0 +1,321 @@
+---
+layout: course
```

Comment:
I think you should be able to omit this now as the layout will come from the 11tydata.js file in the /css/ directory.

Reply:
Sweet. I think we can probably abstract most of this stuff into the data file. 

To answer the avatar URL, shoot for this one: https://assets.codepen.io/174183/2021_Cropped.jpg